index.bs

```diff
@@ -1222,13 +1239,56 @@ dictionary MLSingleInputSupportLimits {
     :: {{MLSupportLimits}} for output operand.
 </dl>
 
+### {{MLContext/destroy()}}  ### {#api-mlcontext-destroy}
+
+The {{MLContext/destroy()}} method can be called to release all resources associated with the context. Any scheduled compute requests will fail.
+
+<details open algorithm>
+<summary>
+    The <dfn method for=MLContext>destroy()</dfn> method steps are:
+</summary>
+    1. If [=this=] [=MLContext/is lost=], then abort these steps.
+    1. For each {{MLGraph}} |graph| where |graph|.{{MLTensor/[[context]]}} equals [=this=]:
+        1. Run the {{MLGraph/destroy()}} method steps for |graph|.
+    1. Run the steps to [=context lost|lose=] [=this=].
```

Comment:
Any reason to do this before destroying the tensors?

Reply:
I was following https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/ml/ml_context.cc;l=150

@mingmingtasd - do you recall if this ordering was intentional or arbitrary?

Reply:
Yes, we'll destroy all the MLTensors and MLGraphs under the MLContext, but `Run the steps to [=context lost|lose=] [=this=]` this step seems unnecessary.